pepper-apis/docs/specification/src/endpoints/user.profile.addresses.verify.yml

```diff
@@ -12,9 +12,62 @@ post:
   tags:
     - Mailing Address
   requestBody:
-    $ref: '../pepper.yml#/components/requestBodies/MailingAddress'
+    required: true
+    content:
+      application/json:
+        schema:
+          allOf:
+            - $ref: '../pepper.yml#/components/schemas/BaseMailAddress'
+            - type: object
+              properties:
+                studyGuid:
+                  type: string
+                  description: if provided, will check address against study rules for any potential issues
   responses:
     200:
-      $ref: '../pepper.yml#/components/responses/MailingAddress'
+      description: a corrected address that's better suited for deliverability, along with any warnings
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '../pepper.yml#/components/schemas/BaseMailAddress'
```

Comment:
We're not dealing with all the properties in a mailing address (e.g. `description`, `validationStatus`, etc) so it's just the basic address.